### DISCUSSION TITLE
Release v0.18.0 Beta

### DISCUSSION THREAD

**Discussion by PrestonN** [FreeTubeApp/FreeTube#2797] [Announcements]
## Features

* Playlists now show a progress bar by @makerio90 in https://github.com/FreeTubeApp/FreeTube/pull/2402
* You can now open up a new window when searching by holding the shift key. Works while clicking on the search arrow or when hitting the enter key. By @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/2427
* You can now open up a new window for videos and channels by right clicking on them by @constraintAutomaton in https://github.com/FreeTubeApp/FreeTube/pull/2027
* Chapters now display within the video player as well as in a new box to jump around in the video by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/2224
* You can now refresh subscriptions, trending, and most popular pages by using the `r` key by @Aiz0 in https://github.com/FreeTubeApp/FreeTube/pull/2689
* The video player now has improved touch functionality. Double tap the sides to go back / forward by a few seconds. The video player also better responds to touch for playing / pausing. By @MarmadileManteater in https://github.com/FreeTubeApp/FreeTube/pull/2719
* Disable http disk cache and implempent in-memory image cache by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/2498

## Fixes

* Fix issue with unsubscribing from the "All Channels" profile by @ChunkyProgrammer in https://github.com/FreeTubeApp/FreeTube/pull/2451
* Fix issue where the minimize window shortcut wouldn't function when viewing a video by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/2486
* Fix an issue where Sponsorblock markers would sometimes not display by @MarmadileManteater in https://github.com/FreeTubeApp/FreeTube/pull/2493
* Video titles now localize properly by @MarmadileManteater in https://github.com/FreeTubeApp/FreeTube/pull/2535
* Fixed an issue where the Invidious list within settings wouldn't update properly when typing by @MarmadileManteater in https://github.com/FreeTubeApp/FreeTube/pull/2601
* Fixed an issue where shortened linked in the description wouldn't work by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/2572
* Tweaked the quality selector in the video player so that it doesn't overflow into the fullscreen button by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/2556
* Fixed an issue where the search bar would close on mobile when using a virtual keyboard by @MarmadileManteater in https://github.com/FreeTubeApp/FreeTube/pull/2667
* The settings page should resize better for mobile views by @MarmadileManteater in https://github.com/FreeTubeApp/FreeTube/pull/2694
* Fixed an issue where clicking on a channel while holding `Ctrl` wouldn't open up a new window when on the watch page by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/2713
* Fix issue where the  `Region for Trending` dropdown wouldn't populate by @MarmadileManteater in https://github.com/FreeTubeApp/FreeTube/pull/2737

## Changes

* Comments now show bold, italics and strikethrough formatting by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/2475
* FreeTube now uses Electron v21 by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/2717
* FreeTube is once again using the upstream ytdl-core module for the local API by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/2510
* Updated yt-trending-scraper, yt-comment-scraper, and yt-channel-info modules for the local API
  * This fixes the recent channel issues due to YouTube updating their layout
* Upcoming live streams now show the time remaining until they will go live by @Aiz0 in https://github.com/FreeTubeApp/FreeTube/pull/2501
* Binary builds have been reduced in size by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/2511 https://github.com/FreeTubeApp/FreeTube/pull/2569
* Number separators should now be properly localized by @ChunkyProgrammer in https://github.com/FreeTubeApp/FreeTube/pull/2557
* FreeTube now remembers if it was fullscreen when closed and will resume in fullscreen next time you open the app by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/2495
* Enabled the Urdu locale by @ArshErgon in https://github.com/FreeTubeApp/FreeTube/pull/2509
* You can now prevent FreeTube from automatically fetching subscriptions during the app startup by @Aiz0 in https://github.com/FreeTubeApp/FreeTube/pull/2632
* The logic for importing subscriptions has been tweaked by @ChunkyProgrammer in https://github.com/FreeTubeApp/FreeTube/pull/2604
* Invidious instances that have the API disabled will no longer show up in the instance list within settings by @MarmadileManteater in https://github.com/FreeTubeApp/FreeTube/pull/2714
* The Most Popular tab will now be hidden if the Invidious API is disabled by @ChunkyProgrammer in https://github.com/FreeTubeApp/FreeTube/pull/2772
* The app menu now has a link to go to the settings page by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/2777

## New Builds

We are now building for Linux armv7l and Windows arm64. We also now have 7zip variants for all portable releases. Check out the new builds over on the website.

## New Contributors
* @MarmadileManteater made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/2483
* @makerio90 made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/2402
* @cappic90 made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/2537
* @Aiz0 made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/2501
* @Jai-JAP made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/2113
* @ArshErgon made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/2509

**Full Changelog**: https://github.com/FreeTubeApp/FreeTube/compare/v0.17.1-beta...v0.18.0-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.18.0-beta'>Release v0.18.0 Beta</a>.</em>

**Comment by PikachuEXE** [comment]
> The app menu now has a link to go to the settings page

This is mainly for the keyboard shortcut

**Comment by PikachuEXE** [comment]
> FreeTube now uses Electron v21

PR should be https://github.com/FreeTubeApp/FreeTube/pull/2717

**Reply by PrestonN** [reply]
Fixed!

**Reply by AvianaDownloadLinux** [reply]
Would a new Freetube official build coming soon this year. it's seem to be a dated at this point

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @AvianaDownloadLinux ,

There are still a couple of things for us to figure out but it shouldn't take long anymore! 

If u cant wait for the upcoming release and want to use FT in its current form, consider to download [the latest nightly build](https://docs.freetubeapp.io/development/nightly-builds).

I know that it doesnt matter because we dont overwrite these files but make sure to make a backup of your [notable FT files](https://docs.freetubeapp.io/usage/data-location/) before u install the latest nightly build.

**Comment by guest32840** [comment]
The new executable freetube.exe is detected as PDM:Trojan.Win32.Generic by Kaspersky. It was detected only after playing a couple videos, which was about 20 minutes after installing it. There was nothing detected in the setup file, freetube-0.18.0-setup-x64.exe. I even tried extracting freetube.exe from the setup file and scanning it, but nothing was detected. It probably has something to do with the behavior of freetube.exe while it is running. Any thoughts?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
@guest32840 nothing to worry about. See https://github.com/FreeTubeApp/FreeTube/issues/261#issuecomment-495623002

**Comment by AndreJesusBrito** [comment]
Its been a while since a new release of FreeTube even though the GitHub its very active.
Any plans for a release? I think that having a shorter release cycle could be beneficial for people using FreeTube as a daily driver (my case) to enjoy new features and bug fixes.

Thanks you for all the great work put in this project.

**Reply by mkalinski** [reply]
Especially since, in the current most recent release, local API is completely broken because of #3402. It's apparently fixed in nightly, and I think this is severe enough to warrant a new release. Please consider.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi there,

There are still a couple of things for us to figure out but it shouldn't take long anymore! 

If u cant wait for the upcoming release and want to use FT in its current form, consider to download [the latest nightly build](https://docs.freetubeapp.io/development/nightly-builds).

I know that it doesnt matter because we dont overwrite these files but make sure to make a backup of your [notable FT files](https://docs.freetubeapp.io/usage/data-location/) before u install the latest nightly build.

**Comment by ExpandingMan** [comment]
I don't suppose there are nightly appimages being built somewhere?  I agree it would be nice to have a bit more of a release cadence on this, seems like a ton of work is getting done all the time but it's quite a while between releases.

**Reply by ChunkyProgrammer** [reply]
Hi,

You can find nightly appimages here: 
https://docs.freetubeapp.io/development/nightly-builds 

You could also try this link (which i find nicer to use but may be ratelimited) 
https://nightly.link/FreeTubeApp/FreeTube/workflows/build/development

**Reply by ExpandingMan** [reply]
awesome, thanks!

**Comment by Aeternus-Sentiens** [comment]
When I downloaded and installed this release (on Windows 10), suddenly I couldn't open Freetube at all. (I would click on it, any of the shortcuts in the start menu or task bar, or even the desktop shortcut, and nothing happened. No system pop-up, nothing.) I tried uninstalling and re-installing, restarting my computer, multiple times. Finally, I uninstalled it and then installed the previous  release (0.17.0) and it worked just fine again. So this release just didn't work at all for me. Not sure why since at least some people seem to be able to get it running.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi there,

Could u try to install [the latest nightly build](https://docs.freetubeapp.io/development/nightly-builds) to check if that solves your issue.

I know that it doesnt matter because we dont overwrite these files but make sure to make a backup of your [notable FT files](https://docs.freetubeapp.io/usage/data-location/) before u install the latest nightly build.
